hoot-test/src/main/cpp/hoot/test/main.cpp

```diff
@@ -330,76 +330,44 @@ void runSingleTest(CppUnit::Test * pTest, QStringList &args, CppUnit::TextTestRe
 
 void populateTests(_TestType t, std::vector<TestPtr> &vTests, bool printDiff, bool hideDisableTests = false)
 {
-  /** This section is a bit verbose but ordering is very important as the order must go as follows:
-   *  Default Registry
-   *  Script Tests
-   *  Named Registries (current, quick, etc.)
-   */
-  switch(t)
```

Comment:
This is no longer the case.  Starting with the parallel unit test work, order was no longer significant.